### PR TITLE
Adding all files for BlaBlaCar case study

### PR DESCRIPTION
**Pull request by kbarnard10**
Adding all files for BlaBlaCar case study: 
- HTML file
- CSS file
- Updated Index file
- Logos and Images

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/5857)

### CODE REVIEW

File: case-studies/blablacar.html [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+<hr>
+
+<section class="section1">
+<div class="cols">
+  <div class="col1">
+    <h2>Challenge</h2>
+        The world’s largest long-distance carpooling community, <a href="https://www.blablacar.com/">BlaBlaCar</a>, connects 40 million members across 22 countries. The company has been experiencing exponential growth since 2012 and needed its infrastructure to keep up. "When you’re thinking about doubling the number of servers, you start thinking, ‘What should I do to be more efficient?’" says Simon Lallemand, Infrastructure Engineer at BlaBlaCar. "The answer is not to hire more and more people just to deal with the servers and installation." The team knew they had to scale the platform, but wanted to stay on their own bare metal servers.
+    <br>
+      <h2>Solution</h2>
```
Comment (Bradamant3) [inline case-studies/blablacar.html]: The layout in the output isn't quite right here. The HTML is at least one culprit -- it looks as though you need one more `<br>` tag, per comparison with other case studies. So you'd have:
`<h2>Challenge</h2>
    The world's largest ...
<br><br>
<h2>Solution</h2>
    ...`

File: case-studies/blablacar.html [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        Opting not to shift to cloud virtualization or use a private cloud on their own servers, the BlaBlaCar team became early adopters of containerization, using the CoreOs runtime <a href="https://coreos.com/rkt">rkt</a>, initially deployed using <a href="https://coreos.com/fleet/docs/latest/launching-containers-fleet.html">fleet</a> cluster manager. Last year, the company switched to <a href="http://kubernetes.io/">Kubernetes</a> orchestration, and now also uses <a href="https://prometheus.io/">Prometheus</a> for monitoring.
```
Comment (Bradamant3) [inline case-studies/blablacar.html]: There's an issue with the link to `fleet` here in the preview, but the HTML looks fine. I'm trying to chase it down.

File: case-studies/blablacar.html (hunk 1 above)
Comment (kbarnard10) [reply case-studies/blablacar.html]: Ah - thanks for catching that! I totally glanced over it. Updated the HTML.

File: case-studies/blablacar.html (hunk 2 above)
Comment (kbarnard10) [reply case-studies/blablacar.html]: This link is working for me. Are you still seeing issues?

**Comment by kbarnard10** [comment]
Any update on this?

**Pull request merged**